simplenodeservice/dynatrace/loadtestingdashboard.json

```diff
@@ -0,0 +1,1015 @@
+                  "name": "dt.entity.host",
+                  "values": [],
+                  "entityDimension": true
+                }
+              ],
+              "sortAscending": false,
+              "sortColumn": true,
+              "aggregationRate": "TOTAL"
+            }
+          ],
+          "resultMetadata": {}
+        },
+        "filtersPerEntityType": {}
+      }
+    },
+    {
+      "name": "Custom chart",
+      "tileType": "CUSTOM_CHARTING",
+      "configured": true,
+      "bounds": {
+        "top": 684,
+        "left": 1330,
+        "width": 342,
+        "height": 152
+      },
+      "tileFilter": {
+        "timeframe": null,
+        "managementZone": null
+      },
+      "filterConfig": {
+        "type": "MIXED",
+        "customName": "Disk used %",
+        "defaultName": "Custom chart",
+        "chartConfig": {
+          "legendShown": true,
+          "type": "TIMESERIES",
+          "series": [
+            {
+              "metric": "builtin:host.disk.usedPct",
+              "aggregation": "AVG",
+              "percentile": null,
+              "type": "LINE",
+              "entityType": "HOST",
+              "dimensions": [
+                {
+                  "id": "0",
+                  "name": "dt.entity.host",
+                  "values": [],
+                  "entityDimension": true
+                },
+                {
+                  "id": "1",
+                  "name": "dt.entity.disk",
+                  "values": [],
+                  "entityDimension": true
+                }
+              ],
+              "sortAscending": false,
+              "sortColumn": true,
+              "aggregationRate": "TOTAL"
+            }
+          ],
+          "resultMetadata": {}
+        },
+        "filtersPerEntityType": {}
+      }
+    },
+    {
+      "name": "Markdown",
+      "tileType": "MARKDOWN",
+      "configured": true,
+      "bounds": {
+        "top": 0,
+        "left": 0,
+        "width": 988,
+        "height": 76
+      },
+      "tileFilter": {
+        "timeframe": null,
+        "managementZone": null
+      },
+      "markdown": "## Application Under Test Insights\n\n[Diagnostics of Test Steps](https://hci34192.live.dynatrace.com/ui/diagnostictools/mda?gtf=l_2_HOURS&gf=all&metric=RESPONSE_TIME&dimension=%7BRequestAttribute:TSN%7D&aggregation=AVERAGE&chart=LINE&servicefilter=0%1E15%11ad4a5402-f253-4d84-ae93-debf5904e8d6)\n- [Service Flow](https://hci34192.live.dynatrace.com/#serviceflow;sci=SERVICE-A8737F1ED40B8C8F) - [Top Web Requests](https://hci34192.live.dynatrace.com/#topglobalwebrequests;sci=SERVICE-A8737F1ED40B8C8F) - [Method Hotspot Analysis](https://hci34192.live.dynatrace.com/#responsetimeanalysis;sci=SERVICE-A8737F1ED40B8C8F) - [Outlier Analysis](https://hci34192.live.dynatrace.com/#responsetimedistribution;sci=SERVICE-A8737F1ED40B8C8F) - [PurePaths](https://hci34192.live.dynatrace.com/#servicecalls;sci=SERVICE-A8737F1ED40B8C8F)"
```

Comment:
This contains a link to your dynatrace tenant - not sure if that was intended?

Reply:
no problem at all